persistent-mysql-haskell/persistent-mysql-haskell.cabal

```diff
@@ -42,7 +42,7 @@ library
                    -- keep the following in sync with @mysql-haskell@ .cabal
                    , io-streams            >= 1.2     && < 2.0
                    , time                  >= 1.5.0
-                   , network               >= 2.3     && < 3.0
+                   , network               >= 2.3     && < 4.0
```

Comment:
Note that `network` follows the PVP and *not* SemVer and therefore this upper bound makes no sense; see also https://pvp.haskell.org/faq/#how-does-the-pvp-relate-to-semantic-versioning-semver